openslides/users/models.py

```diff
@@ -3,12 +3,13 @@
 from random import choice
 
 from django.contrib.auth.hashers import make_password
-from django.contrib.auth.models import (
+from django.contrib.auth.models import (  # noqa
```

Comment:
I used 'isort' to generate the imports. I would like to keep it the way, so this file is ready for the isort check